shoop/core/migrations/0013_address_changes.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django_countries.fields
+import shoop.core.utils.name_mixin
+import django.db.models.deletion
+import shoop.core.models._base
+import shoop.core.fields
+import shoop.utils.models
+
+
+def from_data(cls, data, immutable=False):
+    """
+    Create Address from a data dictionary.
+
+    :type data: dict[str,str]
+    :type immutable: boolean
+    :rtype: Unsaved Address
+    """
+    if immutable:
```

Comment:
What's the point of the `immutable` arg if you're passing in `MutableAddress` and `ImmutableAddress` as `cls`?